core/services/keeper/common.go

```diff
@@ -4,19 +4,22 @@ import (
 	"math/big"
 	"time"
 
-	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common"
 
-	evmtypes "github.com/smartcontractkit/chainlink/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/core/gethwrappers/generated/keeper_registry_wrapper1_3"
```

Comment:
nit: can you club this import with the other wrapper imports

Reply:
Done